### PR TITLE
Add more text around privacy, user experience and camera-in-use indicators

### PR DESCRIPTION
**Pull request by bialpio**
Expand the non-normative text around user consent, call out some specific examples on how it could be granted. Add normative text for camera-in-use indicators (they are required! - this should be in line w/ gUM() API), plus minor edits (feature policy was renamed, call out that "camera" permissions policy is also needed).

Relevant for issues  #10, w3ctag/design-reviews#652.

Pinging @toji, @AdaRoseCannon, @torgo to bring this to your attention.

***
<a href="https://pr-preview.s3.amazonaws.com/immersive-web/raw-camera-access/pull/12.html" title="Last updated on Jul 12, 2022, 8:25 PM UTC (ef171a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/raw-camera-access/12/33f6a73...ef171a5.html" title="Last updated on Jul 12, 2022, 8:25 PM UTC (ef171a5)">Diff</a>

### CODE REVIEW

**Review by toji — APPROVED** [review]
Good updates, very thorough consideration of the privacy concerns. Thanks! A couple of minor style suggestions.

File: index.bs [hunk 1]
```diff
@@ -266,14 +269,51 @@ Privacy & Security Considerations {#privacy-security}
...
+
+</section>
+
+Note: It is the intent of this specification to ensure that the user agent does not encourage app developers to ask for [=camera-access=] feature if the experience they want to provide does not absolutely require it. Due to privacy implications of the feature, user agents are allowed to introduce additional friction when a site asks for access to the camera images, with the hopes that this would incentivize the applications to not ask for the feature needlessly.
+
+Privacy indicators {#privacy-indicators}
+------------------
+
+The user agent SHOULD attempt to display a privacy indicator every time an {{XRSession}} with [=camera-access=] has been created and has not yet [=XRSession/ended=]. If it ever becomes possible to modify a [=XRSession/set of granted features=] of a session after a session has been created, the indicator SHOULD be displayed for at least as long as the [=camera-access=] feature descriptor is [=list/contain|contained=] in the [=XRSession/set of granted features=] of the session.
```
Comment (cynthia) [inline index.bs:303]: "SHOULD attempt to" probably is more spec-texty if it is simplified to "SHOULD" 

That said, any reason why this is not a MUST?

File: index.bs [hunk 2]
```diff
@@ -266,14 +269,51 @@ Privacy & Security Considerations {#privacy-security}
...
+
+<p align="center">
+  <img src="img/permission-mock.png"></img>
+</p>
+
+2. Displaying a series of permission prompts that would take into consideration various levels of user consent needed in order to create a session with the requested and optional features provided by the application to {{XRSystem/requestSession()}} call.
+
+</section>
+
+Note: It is the intent of this specification to ensure that the user agent does not encourage app developers to ask for [=camera-access=] feature if the experience they want to provide does not absolutely require it. Due to privacy implications of the feature, user agents are allowed to introduce additional friction when a site asks for access to the camera images, with the hopes that this would incentivize the applications to not ask for the feature needlessly.
```
[the remark this answers is not in the document]
Comment (bialpio) [reply index.bs:298]: Ack, stay tuned for the update!

File: index.bs (hunk 1 above)
Comment (bialpio) [reply index.bs:303]: Sounds good, I'll switch it to MUST - I was a bit worried about potential form factors where showing the indicator is impossible, but I think we can tighten it now and think about relaxing it if need arises.

**Pull request merged**